doghouse/server/github.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/bradleyfalzon/ghinstallation"
+	"github.com/google/go-github/github"
+)
+
+type NewGitHubClientOption struct {
+	// Required
+	PrivateKey []byte
+	// Required
+	IntegrationID int
+
+	// Either InstallationID OR (RepoOwner AND RepoName) is required.
+	InstallationID int
+	RepoOwner      string
+	RepoName       string
+
+	// Optional
+	Client *http.Client
+}
+
+func NewGitHubClient(ctx context.Context, opt *NewGitHubClientOption) (*github.Client, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewGitHubClient should have comment or be unexported